salesagility/SuiteCRM
# Scheduled Reports: Edit View: Various smaller fixes

## Description

Various smaller fixes for the "Scheduled Reports" edit view page, each in a separate commit.

Broken things:

* Removing emails didn't work
* It defaulted to the advanced cron input instead of the basic one
* The cron input selects didn't get translated to other languages
* Showing it led to PHP errors in the log because of none-existing leftover fields

**Layout before:**
![editview_before](https://user-images.githubusercontent.com/991986/60666351-1c972b80-9e67-11e9-908a-bf430a0c4e5a.png)

**Layout after:**
![editview_after](https://user-images.githubusercontent.com/991986/60666368-2a4cb100-9e67-11e9-8290-230aa455ead3.png)

**Month day select before/after (German):**
![month_before](https://user-images.githubusercontent.com/991986/60666398-3e90ae00-9e67-11e9-8e56-f9b9593d18ec.png)
![month_after](https://user-images.githubusercontent.com/991986/60666402-42bccb80-9e67-11e9-989a-6999b22a6e04.png)

**Week day select before/after (German):**
![week_before](https://user-images.githubusercontent.com/991986/60666454-5bc57c80-9e67-11e9-8f52-f1c5f24986b6.png)
![week_after](https://user-images.githubusercontent.com/991986/60666457-5e27d680-9e67-11e9-898f-ffadceff4503.png)


## Motivation and Context

I'd like my users to create scheduled reports themselves and so I'm trying to make the user
experience better (Also removing emails didn't actually work..)

## How To Test This

* Create a new Scheduled Report record
* The edit view should show the basic cron input with translated strings in one row
* Add an email address and save.
* Re-edit the record and try to remove the email address again, it should work.

## Types of changes

### Final checklist
